e2e/cypress/e2e/loginFlow.cy.js

```diff
@@ -0,0 +1,128 @@
+const nextjsBaseUrl = Cypress.env("nextjsBaseUrl");
+
+describe(`Different ways to open login modal with library with borrowerCheck `, () => {
+  it("Plain login from login button in header", () => {
+    cy.visit(nextjsBaseUrl);
+    cy.consentAllowAll(); //allow cookies
+    cy.get("[data-cy=header-link-login]").should("be.visible").click();
+    cy.get("[data-cy=pickup-search-input]").should("be.visible").type("Val");
+    cy.contains("Valby Bibliotek").click();
+    cy.get("[data-cy=button-log-ind]").should("be.visible");
+  });
+
+  it("Plain login from login button on profile page", () => {
+    cy.visit(nextjsBaseUrl + "/profil");
+    cy.consentAllowAll();
+    cy.get('[class^="ProfileLayout_loginButton__sdKEd"]') //we can have two login buttons on the page, take the one where classname starts with this
```

Comment:
ClassNames er vist ikke konsekvente for React, så jeg tror måske dette kan skabe problemer senere?

Reply:
sletter __sdKEd, tænker at den nok altid tager komponentnavn + parent. hvis det fører til problemer må jeg kigge på det.

Reply:
slettet, har givet den en mere specifikt dataCy